2-projects/1-todo-list/script.js

```diff
@@ -1,25 +1,67 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  todos.forEach((element) => {
+    let liEl = document.createElement("li");
+    liEl.className =
+      "list-group-item d-flex justify-content-between align-items-center";
+
+    liEl.innerHTML = element.task;
+    let spanEl = document.createElement("span");
+    spanEl.className = "badge bg-primary rounded-pill";
+    let iCheckEl = document.createElement("i");
+    iCheckEl.classList.add("fa", "fa-check");
+    iCheckEl.ariaHidden = "true";
+
+    //add line-through to toDo text
+    iCheckEl.addEventListener("click", () => {
```

Comment:
how can you change this function so that it also modifies the `completed` property on the todo in the array of todos?

Reply:
I have added one line of code to change the completed property to true.